server/lib/pbench/process_tb.py

```diff
@@ -25,9 +25,14 @@ def __init__(self, config: PbenchConfig, logger: Logger):
         self.config = config
         self.logger = logger
         # receive_dir -- directory where tarballs are received from agent
-        self.receive_dir = self.get_receive_dir()
+        self.receive_dir = self._get_receive_dir()
+        self.token = self.config.get("pbench-server", "put-token")
+        if not self.token:
+            raise ValueError(
+                "Failed: No value for config option put-token in section pbench-server"
```

Comment:
This is a small thing, but it's really not up to `ProcessTb` to determine what constitutes "failure", so the message here should omit the `Failed:` prefix -- if some outer layer of code wants to declare failure, it can add a suitable prefix -- the code here should just report what happened.